kiwi/kiwi.py

```diff
@@ -29,7 +29,17 @@ def extras(help_, version, options, doc):
     """
     Overwritten method from docopt
 
-    We want to show our own usage for -h|--help
+    Shows our own usage message for -h|--help
+
+    :param bool help_: indicate to show help
+    :param string version: version string
```

Comment:
string -> str

Reply:
Uhm yes ok but I used string everywhere in the code and I think it's not really misleading nor causing any trouble. So if this is not urgent let us keep it

Reply:
Yes, I was splitting hair. :laughing: I agree, it's not something urgent or misleading.